etc/clang-format-all.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+. "$(dirname "${BASH_SOURCE[0]}")/../.evergreen/init.sh"
+
+files="$(find "$LIBMONGOCRYPT_DIR/src" -type f -name '*.c' -o -name '*.h' -o -name '*.cpp' -o -name '*.hpp')"
+readarray -t files <<< "$files"
```

Comment:
Is there an alternative to `readarray` for older bash? The default bash on M1 macBook is:

```
% bash --version
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin21)
```